src/components/Item/ProductDetails.tsx

```diff
@@ -0,0 +1,39 @@
+import { Details, UserStats } from "./ProductDetails.style";
+
+import ProductHead from "./ProductHead";
+import ProductDescription from "./ProductDescription";
+import ProductTag from "./ProductTag";
+import Profile from "./Profile";
+import Likes from "./Likes";
+
+type ProductDetailsProps = {
```

Comment:
불필요한 nesting 이 있는거같아용
item 으로 한번 묶음으로서 오히려 작성해야하는 코드 양이 늘어날 것 같은데 그냥 아래와 같이 작성해도 괜찮지 않을까요??

```suggestion
type ProductDetailsProps = {
    id: number;
    name: string;
    description: string;
    price: number;
    tags: string[];
    ownerNickname: string;
    updatedAt: string;
    favoriteCount: number;
    }

```

Reply:
다른 컴포넌트들도 비슷하게 item 으로 묶여있는 경우가 많네요.
전반적으로 확인 후 수정해주시면 좋을 것 같습니다~